region/maridia/inner-pink/Draygon's Room.json

```diff
@@ -109,6 +109,37 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) - Draygon Alive",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"not": "f_DefeatedDraygon"},
+        "canRiskPermanentLossOfAccess",
+        "Gravity",
+        "Morph",
+        {"or": [
+          "canBeVeryPatient",
+          "Wave",
+          "Plasma"
+        ]},
```

Comment:
I wasnt needing any beams to farm. Or is this to kill the goops while shinecharged - maybe that was why I couldnt get it to work; there were a lot of goops and i just jumped over them.